packages/nartus_connectivity/lib/src/nartus_connectivity_plus_impl.dart

```diff
@@ -12,7 +12,8 @@ class ConnectivityPlusServiceImpl extends ConnectivityService {
   Stream<bool> get onConnectivityChange => _connectivityChange.stream;
 
   /// implementation
-  final StreamController<bool> _connectivityChange = StreamController();
```

Comment:
I edited here because I got error "[StreamBuilder. Bad state: Stream has already been listened to [duplicate]]" when listening Stream in bloc